### ISSUE TITLE
CLASS expressions are case-sensitive when exporting to SLD

### ISSUE THREAD

**Issue by geographika** [mapserver/mapserver#5526]
It seems as though the logical operators in ``CLASS`` ``EXPRESSION``s will only create SLD Filters when they are uppercase. 
For example in the Mapfile below only the second class will have a filter created:

```
  MAP
    SYMBOL
      NAME 'SQUARE'
      TYPE vector
      FILLED true
      POINTS
        0 0
        0 1
        1 1
        1 0
        0 0
      END
    END 
    LAYER
      STATUS ON
      NAME "Test"
      TYPE POINT
      DATA "sample"
      LABELMAXSCALEDENOM 10000
      CLASS
        NAME "Not Working"
        EXPRESSION ( ( [Field] > 0 ) and ( [Field] <= 5 ) )
        STYLE
          SYMBOL "SQUARE"
          COLOR 0 176 80
          SIZE 10
        END
      END
      CLASS
        NAME "Working"
        EXPRESSION ( ( [Field] > 5 ) AND ( [Field] <= 10 ) )
        STYLE
          SYMBOL "SQUARE"
          COLOR 0 176 80
          SIZE 10
        END
      END
    END
  END
```
It looks like this is due to a case-sensitive comparison in https://github.com/mapserver/mapserver/blob/master/mapogcsld.c#L4317

Does MapServer have a case-insensitive string search utility function that could be used here?

**Comment by sdlime** [comment]
@geographika, is this still an issue? --Steve

**Comment by geographika** [comment]
@sdlime - yes this is still an issue in master, and hopefully addressed in #5980. 
All the operators in the [EXPRESSION docs](https://mapserver.org/mapfile/expressions.html#logical-expressions) are uppercase but the Mapfile docs state [The Mapfile is NOT case-sensitive](https://www.mapserver.org/mapfile/#notes). It is only the SLD output that seems to be affected by case-sensitivity - the filters work fine on other features. 

While looking at the code it appears operators such as "lt" in [msSLDGetComparisonValue](https://github.com/mapserver/mapserver/blob/fdad948e08a7a5c6bd3aa0e421aa9c2ec2277897/mapogcsld.c#L4331) are case-sensitive. Should I correct these (allowing "LT") as part of this?
